docs/changelog.md

```diff
@@ -45,6 +45,7 @@
 ## 1.4.0
 - feature: citext type represented as a GraphQL String
 - feature: Support for Postgres 16
-- feature: Support for user defined function
+- feature: Support for user defined functions
```

Comment:
1.4.0 has already been cut and referenced externally so I'll move this to master.

We haven't released 1.4.0 into prod yet though, and this is big improvement, so we can cut 1.4.1